pkg/eraser/eraser.go

```diff
@@ -198,6 +248,24 @@ func main() {
 		os.Exit(1)
 	}
 
+	// read excluded values from excluded configmap
+	data, err = os.ReadFile("/run/eraser.sh/excluded/excluded")
+	if err != nil {
+		if os.IsNotExist(err) {
+			log.Info("excluded configmap does not exist", "error: ", err)
+		} else {
+			log.Info("failed to read excluded values", "error: ", err)
+		}
+	}
+
+	// split images stored as comma separated values
+	vals := strings.Split(string(data), ",")
```

Comment:
As discussed, use a JSON blob in the configmap and unmarshal it here. Offers better format-checking, and the possibility of providing finer-grained controls in the future.